vanilla_first_setup/utils/tests.py

```diff
@@ -0,0 +1,50 @@
+# tests.py
+#
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundationat version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import json
+
+
+class Tests:
+
+    def __init__(self):
+        self.__selected_tests = [""]
+        self.__tests = {}
+        self.__tests_path = "/usr/share/org.vanillaos.FirstSetup/tests.json"
+        self.__selected_tests_path = "/home/$REAL_USER/.local/share/selected_tests.json"
```

Comment:
```suggestion
        self.__selected_tests_path = "/home/$REAL_USER/.local/share/selected_tests.json"
        self.__selected_tests_path = "/home/{}/.local/share/selected_tests.json".format(os.getenv("REAL_USER"))
```
Python cannot resolve environment variables by using `$name`, os.getenv has to be used